layout/formlayout.go

```diff
@@ -33,15 +33,14 @@ func (f *formLayout) countRows(objects []fyne.CanvasObject) int {
 // The width of the label column will be set as the max width value between all the label cells.
```

Comment:
I think the comment for this function should be updated to make it clear that it now returns, in order, the label col width, the content col width, and the row heights. Maybe it should be renamed too like `calculateTableSizes`?